src/integration/pages/homePage/canonicalTests.js

```diff
@@ -14,10 +14,13 @@ export default ({ service, pageData, displayAds }) => {
     runCanonicalAdsTests();
   }
 
-  it('should render the correct number of curations, including most read & radio schedule', () => {
+  it('should render the correct number of curations, including most read, radio schedule & VJ embed', () => {
     const curationsWithSummaries = pageData.curations.filter(
-      ({ summaries, mostRead, radioSchedule }) =>
-        (summaries && summaries?.length > 0) || mostRead || radioSchedule,
+      ({ summaries, mostRead, radioSchedule, vjEmbed }) =>
```

Comment:
I think the name of the variable is `embed`?
```suggestion
      ({ summaries, mostRead, radioSchedule, embed }) =>
```